app/controllers/games_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.create!(game_params.merge(white_player_id: current_user.id))
+    if @game.save
+      redirect_to game_path(@game)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show
+    begin 
+      @game = Game.find(params[:id])
+    rescue
+      if @game.nil?
+        render text: 'Game Not Found', status: :not_found
+      end
+    end
+  end
+
+  def update 
+    @game = Game.find(params[:id])
+    @game.update_attributes(current_user.id)
+    redirect_to game_path(@game)
+  end
+
+  private
+
+  def game_params
+    params.require([:state]).permit(:name)
```

Comment:
This should say `:game` instead of `:state`. I believe it used to be correct, so I'm guessing this is a left-over from all of the test debugging that happened recently.